src/blueprints/tv2/factories/tv2-atem-video-mixer-timeline-factory.ts

```diff
@@ -45,4 +53,52 @@ export class Tv2AtemVideoMixerTimelineFactory implements Tv2VideoMixerTimelineOb
   private getDownstreamKeyerLayerPrefix(): string {
     return Tv2AtemLayer.DOWNSTREAM_KEYER
   }
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  public createDownstreamKeyerFullPilotTimelineObjects(_blueprintConfiguration: Tv2BlueprintConfiguration): TimelineObject[] {
+    return []
+  }
+
+  public createFullPilotTimelineObjects(_blueprintConfiguration: Tv2BlueprintConfiguration, properties: AtemFullPilotTimelineObjectProperties): TimelineObject[] {
+    return [
+      this.createPrimaryMixEffectTimelineObject(Tv2SwitcherMixEffectLayer.PROGRAM, properties),
+      this.createPrimaryMixEffectTimelineObject(Tv2SwitcherMixEffectLayer.CLEAN, properties),
+      this.createNextAuxTimelineObject(properties)
+    ]
+  }
+
+  private createNextAuxTimelineObject(properties: AtemFullPilotTimelineObjectProperties): AtemAuxTimelineObject {
+    return {
+      id: '',
+      enable: { start: 0 },
+      priority: 0,
+      layer: `atem_${Tv2SwitcherAuxLayer.LOOKAHEAD}`,
+      content: {
+        deviceType: DeviceType.ATEM,
+        type: AtemType.AUX,
+        aux: {
+          input: properties.content.input
+        }
+      }
+    }
+  }
+
+  private createPrimaryMixEffectTimelineObject(layer: Tv2SwitcherMixEffectLayer, properties: AtemFullPilotTimelineObjectProperties): AtemMeTimelineObject {
+    const id: string = '' // Todo: get hashId value of 'Tv2SwitcherMixEffectLayer.PROGRAM'
```

Comment:
We don't use the id of TimelineObjects for anything. Just let it be an empty string